debian/README.md

```diff
@@ -0,0 +1,135 @@
+# Debian Packer Templates for MAAS
+
+## Introduction
+
+The Packer templates in this directory creates Debian images for use with MAAS.
+
+## Prerequisites (to create the image)
+
+* A machine running Debian 18.04+ with the ability to run KVM virtual machines.
```

Comment:
what is this Debian 18.04+ of which you speak?

Reply:
Fixing...